### PR TITLE
JAMES-1758 Add DOCKER_HOST dependent profiles

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by rouazana** [comment]
Sorry, I don't understand why you deactivate entirely projects if no DOCKER_HOST, instead of ignoring some specific tests.

**Comment by aduprat** [comment]
I don't deactivate the entire project.
I have just added a `maven-surefire-plugin` configuration in order to exclude such tests.

**Comment by rouazana** [comment]
So I read to fast

**Comment by chibenwa** [comment]
Mmmm....
So what happen if I have a docker UNIX socket on my post but do not wish to play the tests over http ?

**Comment by rouazana** [comment]
:+1:

**Comment by aduprat** [comment]
That's why i added the comment: `depending on your docker configuration`

File: README.txt [hunk 1]
```diff
@@ -84,6 +84,10 @@ as it is needed by the container that will run James.
 - SHA1 (optional): is the given git SHA1 of the james-project repository to build or trunk if none.
 - -s option: given tests will not be played while building. Not specifying means play tests.
 
+Some tests needs a DOCKER_HOST environment variable in order to be played, they will be ignored if you don't provide this variable.
+If you wish to play them, you may use a command like the following (depending on your docker configuration):
+$ docker run --env DOCKER_HOST=tcp://172.17.0.1:2376 -v $PWD/.m2:/root/.m2 -v $PWD:/origin -v $PWD/dockerfiles/run/spring/destination:/destination -t james/project SHA1
```
Comment (mbaechler) [inline README.txt]: you should document the unix socket mount strategy to play the tests on a out of the box docker install

Comment (aduprat) [inline README.txt]: done

**Comment by rouazana** [comment]
:+1:

**Comment by mbaechler** [comment]
:+1:

**Pull request merged**